### PR TITLE
Make sure labels become blue when parent of any level has focus

### PR DESCRIPTION
**Pull request by adrienDog**

### CODE REVIEW

**Comment by adrienDog** [comment]
@StevePole @oliverviljamaa

**Comment by StevePole** [comment]
Why is it necessary?

**Comment by adrienDog** [comment]
@StevePole  Transferflow and user app have grey labels on focus because their labels are not direct descendants of the `.focus` :(
transferwise

**Comment by StevePole** [comment]
Ok, :+1:

File: less/forms.less [hunk 1]
```diff
@@ -729,7 +729,7 @@ input[type="checkbox"] {
     }
 }
 :not(.has-error).focus {
-    > label {
```
Comment (oliverviljamaa) [inline less/forms.less:732]: I don't think we need to keep the regular `label` styling here, can improve specificity by just having `.control-label`.

**Pull request merged**